app/adapters/content.py

```diff
@@ -101,10 +105,12 @@ def _create_table(self) -> None:
                 title TEXT,
                 url TEXT,
                 markdown TEXT,
+                breadcrumbs TEXT,
                 updated_at TIMESTAMP NOT NULL DEFAULT NOW()
             );
             """,
             "ALTER TABLE notion_contents ADD COLUMN IF NOT EXISTS title TEXT;",
+            "ALTER TABLE notion_contents ADD COLUMN IF NOT EXISTS breadcrumbs TEXT;",
```

Comment:
Why not json or jsonb type?

Reply:
I'm saving in text format, not the `json` return that comes from the notion api
it might be interesting to save all json nodes, but that would fall into a new feature